src/main/kotlin/no/nav/familie/ef/sak/behandling/grunnbelop/GOmregningService.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.ef.sak.behandling.grunnbelop
+
+import no.nav.familie.ef.sak.behandling.BehandlingRepository
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.springframework.stereotype.Service
+import java.util.UUID
+
+@Service
+class GOmregningService(val behandlingRepository: BehandlingRepository,
+                        val taskRepository: TaskRepository) {
+
+    fun opprettGOmregningTaskForBehandlingerMedUtdatertG(): Int {
+        //val behandlingIds = behandlingRepository.findBehandlingerMedUtdatertG()
```

Comment:
Dere har sikkert en plan her :) 
Har dere vurdert behandlingId vs fagsakId? 
Tenker, hvis man plukker ut alle fagsaker som har en andel som har TOM > 1 mai, så kan man opprette tasks for disse, og så kan tasken vurdere om det skal g-omregnes eller ikke, eks beroende på om siste er iverksatt eller ikke